Source/Plugins/OrientationAnalysis/FilterParameters/EbsdMontageImportFilterParameter.h

```diff
@@ -44,8 +44,6 @@
 
 #include "OrientationAnalysis/FilterParameters/EbsdMontageListInfo.h"
 
-using EbsdMontageListInfo_t = EbsdMontageListInfo;
-
 #if 0
```

Comment:
Remove the whole #if 0 code block if the code contained in it is not going to be used any more.